src/main/java/io/fabric8/maven/docker/service/BuildXService.java

```diff
@@ -63,7 +63,7 @@ protected <C> void useBuilder(ProjectPaths projectPaths, ImageConfiguration imag
         BuildDirs buildDirs = new BuildDirs(projectPaths, imageConfig.getName());
 
         Path configPath = getDockerStateDir(imageConfig.getBuildConfiguration(),  buildDirs);
-        List<String> buildX = Arrays.asList("docker", "--config", configPath.toString(), "buildx");
+        List<String> buildX = Arrays.asList("docker", "buildx");
```

Comment:
@robfrank : Sorry for the late reply, what was the value of `configPath` when you were debugging this issue?